zboxcore/sdk/filerefsworker.go

```diff
@@ -106,13 +114,15 @@ func (o *ObjectTreeRequest) getFileRefs(oTR *oTreeResponse, bUrl string) {
 	}
 	oResult := ObjectTreeResult{}
 	ctx, cncl := context.WithTimeout(o.ctx, time.Second*30)
+	respStatusCode := 500
```

Comment:
We should use 200 as default code and set it to `oTR` after calling `zboxutil.HttpDo()` successfully.